### DISCUSSION TITLE
What do you think about Ouverture Maps dataset ?

### DISCUSSION THREAD

**Discussion by NovaliX-Dev** [organicmaps/organicmaps#5636] [Ideas]
I came across this new project, by the Linux Foundation and many others compagnies, which has the goal to propose a more precise and complete dataset than OSM (From what i've understand). It will be open as well.

The project is still very young thought, so things can change. But maybe, when it will be mature enough, can we implement it into Organic Maps ?

Here is the website link : https://overturemaps.org/
https://github.com/OvertureMaps

**Comment by hb0nd** [comment]
https://bdon.github.io/overture-tiles/places.html#1.08/0/0 you can look at the data on this slippymap

i'm not impressed at all... POIs are out of date, in the completely wrong place & there's a lot of non public-facing business POI. You can read further discussions on the quality here: https://community.openstreetmap.org/t/overturemaps-org-big-businesses-osmf-alternative/

edit:
a new visualisation of overture data: https://msbarry.github.io/planetiler-overture-demo
at least in my area, it's literally just OSM data but with worse POI

**Reply by NovaliX-Dev** [reply]
I guess i didn't search enough about this young foundation before asking up.

It seems not to be as good as i though it would.

Thanks for the clarification.
